raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -72,6 +72,12 @@ contract TokenNetwork is Utils {
         uint256 nonce;
     }
 
+    enum ChannelState {
+        Gone,    // 0 = non-existent or settled
```

Comment:
Maybe `None` or `NonExistent` is better than `Gone`?

Reply:
Sure, will change to NonExistent